bokehjs/src/coffee/api/typings/models/plots.d.ts

```diff
@@ -3,6 +3,11 @@ declare namespace Bokeh {
         plot?: Plot;
     }
 
+    export interface PlotView extends View<Plot> {
+        resize_width_height(use_width: boolean, use_height: boolean, maintain_ar?: boolean,
+                            width?: number, height?: number): void;
+    }
+
```

Comment:
@almarklein, btw., during experimentation phase you don't actually need static APIs, so you can easily proceed with something like:
```
let view: Bokeh.View<Plot> = make_plot(...);
(view as any).resize_width_height(...);
```